### DISCUSSION TITLE
Why non `<div>`'s allowed in `page-footer`?

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#6340] [Q&A] [answered]
While creating a website project, I noted that the `left`, `right`, `center` suboptions of the `page-footer` only seem to accept `<span>`'s and not `<div>`'s.

If I use
```yml
page-footer: |
    <div>abc</div>
```
I don't get anything in the footer: 
```html
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center"></div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>
```

Only when I use `<span>` instead of `<div>` does it "correctly" produce:
```html
<div class="nav-footer-center"><span>abc</span></div>
```

Why is this so?

**Comment by cderv** [comment]
Not sure why, but if you add something after the div (like text or something else), it shows. 

````yaml
  page-footer: |
      <div>abc</div>
      Content
````

**Reply by mcanouil** [reply, marked as answer]
Completely related to:

- <https://github.com/quarto-dev/quarto-cli/issues/5604>

CC. @cscheid

**Reply by mcanouil** [reply]
@dense-set were you using 1.3 or earlier version?

**Reply by dense-set** [reply]
No, v1.3.433.

**Reply by mcanouil** [reply]
So you need 1.4 pre-release.
